go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -304,6 +304,10 @@ func syncServices(services []interface{}, inport, gwBridge string, nodeIP *net.I
 			pair := strings.Split(key, "_")
 			protocol, port := pair[0], pair[1]
 			if externalIP == "" {
+				// Check if this is a host tcp port flow for kapi or etcd 6443, 2379, 2380
+				if (port == "6443" || port == "2379" || port == "2380") && (protocol == "tcp" || protocol == "tcp6") {
```

Comment:
this is too brittle :smiley:
We know that we have these problems now but this won't solve the problems for other pods that run with hostNetwork: true. 
I mean, users will hit packet drops in their deployments with hostNetwork:true for everything that is not listening on those ports, and is common to put apps that need more performance this way